AGENTS.md

```
@@ -790,8 +796,38 @@ touch of an unconfirmed repo:
-     `git ls-remote origin <lane>/<area>-<desc>-<issue>` must return **your** sha. **Compare the tip —
+   - Push **without force**, then **verify the remote tip of the branch you actually pushed** is
+     yours (for a fresh claim that is `<lane>/<area>-<desc>-<issue>`; for a takeover it is the
+     `…-<issue>-2` / `-3` / … name you just created — never re-check the abandoned base claim name).
+     **Resolve the claim ref from the issue number**, not an assumed exact stem — a takeover suffix
+     means the name you pushed is not the one you would have guessed:
+     ```sh
+     # This lane's claim ref(s) for issue N (base or -<n>-k takeover):
+     git ls-remote --heads origin \
+       | awk -v lane="<lane>" -v n="<issue>" \
+           '$2 ~ "^refs/heads/" lane "/.+" && $2 ~ "-" n "(-[0-9]+)?$" {print $1, $2}'
+     ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)